docs/src/understand/sft.rst

```diff
@@ -70,94 +74,674 @@ Prerequisites
+`TURN` will connect to the `SFT` server.
+In the unlikely scenario where no UDP is allowed whatsoever or `SFT` servers may not be able to reach
+the Restund servers that clients are using to make themselves reachable, an `SFT` server itself can
 also choose to proxy itself by a Restund server, which could be different from the Restund servers
-used by clients (see *TURN discovery* flag).
+used by clients (see *`TURN` discovery* flag).
 
-The SFT may need to receive and send traffic over UDP and TCP on a wide range of ports.
-Due to the fact that Kubernetes services do not support setting port ranges, and Kubernetes pods not being publicly routable (at least in IPv4) we require the SFT pods to run in `hostNetwork` mode and the pod will bind directly to the default interface of the node.
+The `SFT` may need to receive and send traffic over UDP and TCP on a wide range of ports.
+Due to the fact that Kubernetes services do not support setting port ranges, and Kubernetes pods not being publicly routable (at least in IPv4) we require the `SFT` pods to run in `hostNetwork` mode and the pod will bind directly to the default interface of the node.
 
-Due to this `hostNetwork` limitation only one SFT instance can run per node so if you want to scale up your SFT deployment you will need to increase the amount of kubernetes nodes in your cluster.
+Due to this `hostNetwork` limitation only one `SFT` instance can run per node so if you want to scale up your `SFT` deployment you will need to increase the amount of kubernetes nodes in your cluster.
 
-As a rule of thumb you will need 1vCPU of compute per 50 participants. SFT will utilise multiple cores. You can use this rule of thumb to decide how many kubernetes nodes you need to provision.
+As a rule of thumb you will need 1vCPU of compute per 50 participants. `SFT` will utilise multiple cores. You can use this rule of thumb to decide how many kubernetes nodes you need to provision.
 
 For more information about capacity planning and networking please refer to the `technical documentation <https://github.com/wireapp/wire-server/blob/eab0ce1ff335889bc5a187c51872dfd0e78cc22b/charts/sftd/README.md>`__
 
+.. _sft-protocol:
+
+Protocol 
+--------
+
+This section provides a high-level description of the `SFT` protocol.
+
+Calling
+~~~~~~~
+
+Calling in Wire comes in two flavors: **one-to-one calls** and **conference calls**.
+
+One-to-one calls are calls between two clients, whereas conference calls can host more than two clients.
+
+Both flavors have the same technological foundation and heavily rely on `WebRTC <https://webrtc.org/>`__ for media encoding/decoding, encryption and media routing.
+
+Conference calls use an additional server-side component (Selective Forwarding `TURN` Server) as well as an additional encryption method.
+
+Call signaling
+..............
+
+All calls are initiated through the `E2EE` session.
```

Comment:
First use of E2EE, consider `End to End Encryption (E2EE)` instead

Reply:
fixed.